app/util/conf.py

```diff
@@ -0,0 +1,42 @@
+import yaml
+from util.project_paths import JIRA_YML, CONFLUENCE_YML
+
+
+def read_yml_file(file):
+    with file.open(mode='r') as file:
+        return yaml.load(file, Loader=yaml.FullLoader)
+
+
+class JiraSettings:
+
+    def __init__(self):
+        obj = read_yml_file(JIRA_YML)
+        self.hostname = obj['settings']['env']['application_hostname']
+        self.protocol = obj['settings']['env']['application_protocol']
+        self.port = obj['settings']['env']['application_port']
+        self.postfix = obj['settings']['env']['application_postfix'] or ""
+        self.admin_login = obj['settings']['env']['admin_login']
+        self.admin_password = obj['settings']['env']['admin_password']
+        self.concurrency = obj['settings']['env']['concurrency']
+
+    @property
+    def server_url(self):
+        return f'{self.protocol}://{self.hostname}:{self.port}{self.postfix}'
+
+
+class ConfluenceSettings(JiraSettings):
+
+    def __init__(self):
+        super().__init__()
+        obj = read_yml_file(CONFLUENCE_YML)
+        self.hostname = obj['settings']['env']['application_hostname']
+        self.protocol = obj['settings']['env']['application_protocol']
+        self.port = obj['settings']['env']['application_port']
+        self.postfix = obj['settings']['env']['application_postfix'] or ""
+        self.admin_login = obj['settings']['env']['admin_login']
+        self.admin_password = obj['settings']['env']['admin_password']
+        self.concurrency = obj['settings']['env']['concurrency']
+
+
+JIRA_SETTINGS = JiraSettings()
```

Comment:
What do you think to make these vars be shorter just `JIRA` and `CONFLUENCE`?

Reply:
makes sense

Reply:
but JIRA_YML_CONFIG is too large as for me, currently, we do not have other settings except yml, maybe JIRA_CONFIG or JIRA_CONF is enough

Reply:
If shorten Confluence, it will be looking like CONF_CONF.
Seems to be nice )))